File: flutter_control_neumaticos/lib/models/usuario.dart
class Usuario {
  final String nombres;
  final String apellidos;
  final String correo;
  final String clave;
  final int perfil;
  final int estado;
  final int bodega;

  Usuario({
    required this.nombres,
    required this.apellidos,
    required this.correo,
    required this.clave,
    required this.perfil,
    required this.estado,
    required this.bodega,
  });

  Map<String, dynamic> toJson() {
    return {
      'nombres': nombres,
      'apellidos': apellidos,
      'correo': correo,
      'clave': clave,
      'codigO_PERFIL': perfil,
      'coD_ESTADO': estado,
      'iD_BODEGA': bodega,
    };
  }
}

File: api_control_neumaticos/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace api_control_neumaticos.Models
{
    public partial class Usuario
    {
        public int IdUsuario { get; set; }

        public string Nombres { get; set; } = "";

        public string Apellidos { get; set; } = "";

        public required string Correo { get; set; }

        public required string Clave { get; set; }

        public int CodigoPerfil { get; set; }

        public int CodEstado { get; set; }

        // Nueva propiedad
        public int ID_BODEGA { get; set; }

        [ForeignKey("ID_BODEGA")]
        public virtual Bodega Bodega { get; set; } = null!;  

        public DateTime? FechaClave { get; set; }

        public int IntentosFallidos { get; set; }

        public virtual ICollection<SolicitudCorreos> SolicitudesEnviadas { get; set; } = new List<SolicitudCorreos>();
    }
}
